apps/docs/src/utils/jsdocLinkToMarkdown.ts

```diff
@@ -0,0 +1,17 @@
+export const jsdocLinkToMarkdown = (jsdocLink: string) => {
+  if (!jsdocLink.includes('@link')) {
+    return jsdocLink
+  }
+  const textBefore = jsdocLink
+    .substring(0, jsdocLink.indexOf('{@link'))
+    .replace(/@\S*/g, '')
+  const url = jsdocLink.substring(
+    jsdocLink.indexOf('http'),
+    jsdocLink.lastIndexOf('/'),
```

Comment:
```suggestion
    jsdocLink.lastIndexOf('|') + 1,
```
Vad tror du om att titta på den där istället? Länkarna behöver ju inte sluta med "/"

Reply:
Sant, det är snyggare

Reply:
Problemet är att det inte går för | är escaped på något sätt

Reply:
Dessutom måste länkarna inte sluta med | heller. Så om vi har som praxis att sluta en länk med `/` så funkar det här och då är vi hemma. Kan finnas fler edge cases men måste kanske inte fånga allt här, inte affärskritiskt

Reply:
Aha, vad lustigt!

Reply:
Eh jag kan inte lösa det här 100% om jag inte lägger ett par timmar till. Ska vi kalla det för utrymme till förbättring.. ? Kan regexpa en url och jobba därifrån men inte få ut `|` vilket gör det svårt att veta om det ens finns en länktext. Vi har bara max en handfull såna här case

Reply:
Japp, vi kör så får vi komma tillbaka till det :)